src/Updater/StdOutput.php

```diff
@@ -2,15 +2,34 @@
 
 namespace my127\Workspace\Updater;
 
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\StringInput;
+use Symfony\Component\Console\Output\ConsoleOutput;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
 class StdOutput implements Output
 {
+    /** @var OutputInterface */
+    private $output;
+
+    public function __construct()
+    {
+        $this->output = new SymfonyStyle(new StringInput(''), new ConsoleOutput());
+    }
+
     public function infof(string $info, ...$args): void
     {
-        echo sprintf($info, ...$args) . PHP_EOL;
+        $this->output->writeln('<info>' . sprintf($info, ...$args) . '</info>');
```

Comment:
hm, why do this here at all? why not just add `writeln` ?

Reply:
ah ok, so this is kindof a "progress" class. it would be good to get the ConsoleOutput from the DI container and have it respect `--no-ansi` etc

i often inject a logger in situations like this.

Reply:
semantics here I think, it's logging progress to stdout, good point on the `--no-ansi` but is it really needed?

Reply:
i don't really mean to do this _here_. But it doesn't make sense to have everything define it's own output methods.

What if you want to add a `--silent` flag?

Using a logger is a good, agnostic, solution for stuff like this as the code is then reusable (while sacrificing on CLI bling). But the bling can be added at the "controller" level (i.e. in Symfony the `Command`)

Reply:
no, just commenting :)